OPAL/ba/src/main/scala/org/opalj/ba/package.scala

```diff
@@ -1129,6 +1129,14 @@ package object ba { ba ⇒
                     }
                 ))
 
+            case br.PermittedSubclasses.KindId ⇒
+                val br.PermittedSubclasses(subclasses) = attribute
+                val attributeNameIndex = CPEUtf8(bi.PermittedSubclassesAttribute.Name)
+                val CPsubclasses = subclasses.map(CPEClass(_, false))
```

Comment:
Please use proper camelCase. Also, looking at the other cases, I think it should be named ...Indices, right?